{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# OpenMS Glossary\n",
    "\n",
    "A glossary of common terms used throughout OpenMS documentation.\n",
    "\n",
    "peptide-spectrum match PSM A method used in proteomics to identify\n",
    "proteins from a complex mixture. Involves comparing the mass spectra of\n",
    "peptide fragments generated from a protein sample with a database of\n",
    "predicted spectra, in order to identify the protein that produced the\n",
    "observed peptides.\n",
    "\n",
    "LC-MS LCMS\n",
    "[Liquid chromatography](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-liquid-chromatography)-coupled\n",
    "mass spectrometry.\n",
    "\n",
    "LC-MS/MS liquid chromatography coupled\n",
    "[tandem mass spectrometry](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-tandem-mass-spectrometry)\n",
    "See\n",
    "[LC-MS](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-LC-MS)\n",
    "and\n",
    "[MS2](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-MS2).\n",
    "\n",
    "liquid chromatography LC An analytical technique used to separate\n",
    "molecules of interest.\n",
    "\n",
    "FASTA  \n",
    "A text-based format for representing nucleotide or amino acid sequences.\n",
    "\n",
    "C18 octadecyl Octadecyl (C18) is an alkyl radical C(18)H(37) derived\n",
    "from an octadecane by removal of one hydrogen atom.\n",
    "\n",
    "ESI electrospray ionization Electrospray ionization (ESI) is a technique\n",
    "used in MS to produce ions.\n",
    "\n",
    "TOF time-of-flight Time-of-flight (TOF) is the time taken by an object,\n",
    "particle or wave (be it acoustic, electromagnetic, e.t.c) to travel a\n",
    "distance through a medium. TOF analyzers can obtain good, but not\n",
    "ultra-high resolution, such as :term:\\`orbitrap\\`s.\n",
    "\n",
    "quadrupole  \n",
    "A mass filter allowing one mass channel at a time to reach the detector\n",
    "as the mass range is scanned. A low resolution MS analyzer.\n",
    "\n",
    "orbitrap  \n",
    "In MS, an ion trap mass analyzer consisting of an outer barrel-like\n",
    "electrode and a coaxial inner spindle-like electrode that traps ions in\n",
    "an orbital motion around the spindle. An ultra-high resolution MS\n",
    "analyzer, capable of resolving fine-isotope structure.\n",
    "\n",
    "Mass Spectrometry MS An analytical technique to measure the mass over\n",
    "charge (m/z) ratio of ions along with their abundance. This gives rise\n",
    "to a mass spectrum (with m/z on the x-axis and abundance on the y-axis).\n",
    "\n",
    "mass spectrum  \n",
    "A visual or numerical representation of a measurement from an MS\n",
    "instrument. A spectrum contains (usually many) pairs of\n",
    "mass-over-charge(m/z)+intensity values.\n",
    "\n",
    "MS1  \n",
    "Mass spectra of a sample where only precursor ions (i.e. no fragment\n",
    "ions) can be observed. Usually MS1 spectra are recorded to select\n",
    "targets for MS2 fragmentation.\n",
    "\n",
    "MS2 MS/MS tandem mass spectrometry Tandem MS is a technique where two or\n",
    "more mass analyzers are coupled together using an additional, usually\n",
    "destructive, reaction step to generate fragment ions which increases\n",
    "their abilities to analyse chemical samples.\n",
    "\n",
    "MS3  \n",
    "Multi-stage MS\n",
    "\n",
    "CID collision-induced dissociation Collision-induced dissociation is an\n",
    "MS technique to induce fragmentation of selected ions in the gas phase,\n",
    "which are subjected to a subsequent measurement (see\n",
    "[MS2](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-MS2)).\n",
    "\n",
    "TOPP  \n",
    "'TOPP - The OpenMS PiPeline' is a pipeline for the analysis of HPLC-MS\n",
    "data. It consists of several small applications that can be chained to\n",
    "create analysis pipelines tailored for a specific problem. See\n",
    "[TOPP tools](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-TOPP-tools).\n",
    "\n",
    "MSGFPlusAdapter  \n",
    "Adapter for the MS-GF+ protein identification (database search) engine.\n",
    "More information is available in the [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_MSGFPlusAdapter.html).\n",
    "\n",
    "LuciphorAdapter  \n",
    "Adapter for the LuciPHOr2: a site localisation tool of generic\n",
    "post-translational modifications from tandem mass spectrometry data.\n",
    "More information is available in the [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_LuciphorAdapter.html).\n",
    "\n",
    "TOPP tools  \n",
    "OpenMS provides a number of applications (executable files) that are\n",
    "chainable in a pipeline/script and each process MS data. These tools are\n",
    "subdivided into different categories, such as 'File Handling' or\n",
    "'Peptide Identification'. All\n",
    "[TOPP](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-TOPP)\n",
    "tools are described in the [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_documentation.html).\n",
    "\n",
    "UTILS  \n",
    "\\[deprecated since OpenMS 3.1\\] Besides\n",
    "[TOPP tools](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-TOPP-tools),\n",
    "OpenMS offers a range of other tools. They are not included in\n",
    "[TOPP](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-TOPP)\n",
    "as they are not part of typical analysis pipelines. Since OpenMS 3.1 all\n",
    "UTILS are\n",
    "[TOPP tools](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-TOPP-tools)\n",
    "under the 'Utilities' category.\n",
    "\n",
    "TOPPView  \n",
    "TOPPView is a viewer for MS and HPLC-MS data and shipped with every\n",
    "OpenMS release.\n",
    "\n",
    "nightly snapshot  \n",
    "Untested installers and containers which are created regularly between\n",
    "official releases and reflect the current development state.\n",
    "\n",
    "MascotAdapter  \n",
    "Used to identify peptides in\n",
    "[MS2](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-MS2)\n",
    "spectra. Read more about MascotAdapter in the [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_MascotAdapter.html).\n",
    "\n",
    "high performance liquid chromatography HPLC In high performance liquid\n",
    "chromatography (HPLC), analytes are dissolved in a pressurized solvent\n",
    "(mobile phase) and pumped through a solid adsorbent material (stationary\n",
    "phase) packed into a capillary column. Physicochemical properties of the\n",
    "analyte determine how strongly it interacts with the stationary phase.\n",
    "\n",
    "mzML mzml The mzML format is an open, XML-based format for mass\n",
    "spectrometer output files, developed by the Proteomics Standard\n",
    "Initiative (PSI) with the full participation of vendors and researchers\n",
    "in order to create a single open format that would be supported by all\n",
    "software.\n",
    "\n",
    "mzData mzdata mzData was the first attempt by the Proteomics Standards\n",
    "Initiative (PSI) from the Human Proteome Organization (HUPO) to create a\n",
    "standardized format for MS data. This format is now deprecated, and\n",
    "replaced by mzML.\n",
    "\n",
    "mzXML mzxml mzXML is an open data format for storage and exchange of\n",
    "mass spectroscopy data, developed at the SPC/Institute for Systems\n",
    "Biology. This format is now deprecated, and replaced by mzML.\n",
    "\n",
    "ProteoWizard  \n",
    "ProteoWizard is a set of open-source, cross-platform tools and libraries\n",
    "for proteomics data analyses. It provides a framework for unified MS\n",
    "data file access and performs standard chemistry and LCMS dataset\n",
    "computations.\n",
    "\n",
    "PepNovo  \n",
    "PepNovo is a de\n",
    "[de novo peptide sequencing](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-de-novo-peptide-sequencing)\n",
    "algorithm for\n",
    "[MS2](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-MS2)\n",
    "spectra.\n",
    "\n",
    "de novo peptide sequencing  \n",
    "A peptideâ€™s amino acid sequence is inferred directly from the precursor\n",
    "peptide mass and tandem mass spectrum\n",
    "([MS2](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-MS2)\n",
    "or\n",
    "[MS3](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-MS3))\n",
    "fragment ions, without comparison to a reference proteome.\n",
    "\n",
    "TOPPAS  \n",
    "An assistant for GUI-driven\n",
    "[TOPP](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-TOPP)\n",
    "workflow design, build into OpenMS. See <span class=\"title-ref\">TOPPAS\n",
    "tutorial\n",
    "\\<https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPPAS_tutorial.html\\></span>\n",
    "for details.\n",
    "\n",
    "KNIME  \n",
    "An advanced workflow editor which OpenMS provides a plugin for.\n",
    "\n",
    "SILAC stable isotope labeling with amino acids in cell culture Stands\n",
    "for Stable isotope labeling using amino acids in cell culture.\n",
    "\n",
    "iTRAQ  \n",
    "Isobaric tags for relative and absolute quantitation (iTRAQ) is a MS\n",
    "based multiplexing technique designed to identify and quantify proteins\n",
    "from different samples in one single measurement.\n",
    "\n",
    "TMT  \n",
    "Tandem Mass Tag (TMT) is a MS based multiplexing technique designed to\n",
    "identify and quantify proteins from different samples in one single\n",
    "measurement.\n",
    "\n",
    "SRM  \n",
    "Selected reaction monitoring (SRM) is a MS technique for targeted small\n",
    "molecule analysis.\n",
    "\n",
    "SWATH  \n",
    "Sequential acquisition of all theoretical fragment ion spectra (SWATH)\n",
    "uses parially overlapping MS2 scans with wide isolation windows to\n",
    "capture all fragment ions in a data independent analysis (DIA).\n",
    "\n",
    "OpenMS API  \n",
    "A C++ interface that allows developers to use OpenMS core library\n",
    "classes and methods.\n",
    "\n",
    "feature features A feature, in the OpenMS terminology, subsumes all m/z\n",
    "signals originating from a single compound at a certain charge state.\n",
    "This includes the isotope pattern and usually spans multiple spectra in\n",
    "retention time (the elution profile).\n",
    "\n",
    "feature maps feature map A feature map is a collection of\n",
    "[feature`s identified from a single experiment.\n",
    "    One feature map usually contains many features. OpenMS represents a feature map using the class `FeatureMap ](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1FeatureMap.html)\\_.\n",
    "\n",
    "consensus features consensus feature Features from replicate experiments\n",
    "with similar retention times and m/z values are linked and considered a\n",
    "consensus feature. A consensus feature contains information on the\n",
    "common retention time and m/z values as well as intensities for each\n",
    "sample. OpenMS represents a consensus feature using the class\n",
    "[ConsensusFeature](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1ConsensusFeature.html).\n",
    "\n",
    "consensus maps consensus map A consensus map is a collection of\n",
    "[consensus features](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-consensus-features)\n",
    "identified from mass spectra across replicate experiments, usually by\n",
    "combining multiple\n",
    "[feature maps](https://pyopenms.readthedocs.io/en/latest/user_guide/glossary.html#term-feature-maps).\n",
    "One consensus map usually contains many consensus features. OpenMS\n",
    "represents a consensus map using the class\n",
    "[ConsensusMap](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1ConsensusMap.html)."
   ],
   "id": "2847e812-15b3-472d-93a7-c3d7eaf0d82a"
  }
 ],
 "nbformat": 4,
 "nbformat_minor": 5,
 "metadata": {}
}
